lib/integrations/awsoidc/idp_iam_config.go

```diff
@@ -83,6 +87,33 @@ type IdPIAMConfigureRequest struct {
 	fakeThumbprint string
 }
 
+// PolicyPreset defines a preset policy type for the AWS IAM role
+// created by the Teleport AWS OIDC integration.
+type PolicyPreset string
+
+const (
+	// PolicyPresetUnspecified specifies no preset policy to apply.
+	PolicyPresetUnspecified PolicyPreset = ""
+	// PolicyPresetAWSIdentityCenter specifies poicy required for the AWS identity center integration.
+	PolicyPresetAWSIdentityCenter PolicyPreset = "aws-identity-center"
+)
+
+// ErrAWSOIDCInvalidPolicyPreset is issued if provided policy preset
+// value is not supported.
+var ErrAWSOIDCInvalidPolicyPreset = &trace.BadParameterError{
+	Message: "--preset-policy defines an unknown preset value",
```

Comment:
It would be nice but it adds a burden for us to update the message every time a new preset will be added and easy to overlook. The message is clear enough for user to reference in docs.